superagent/superagent-tests.ts

```diff
@@ -302,4 +303,13 @@ request
 request
   .get('/search')
   .then((response) => {})
-  .catch((error) => {}); 
\ No newline at end of file
+  .catch((error) => {});
+
+// Requesting binary data.
+// adapted from: https://github.com/visionmedia/superagent/blob/v2.0.0/test/client/request.js#L110
+request
+  .get('/blob')
+  .responseType('blob')
+  .end(function(err, res){
+    assert.deepEqual(res.xhr.response instanceof Blob, true);
```

Comment:
Can do. I was just trying to keep it as close to the original test source as possible.